src/utils.js

```diff
@@ -53,7 +53,8 @@ export async function readPackage() {
   const packageConfig = await require(absConfigPath)
   const playwright =
     checkDependencies(packageConfig.dependencies) ||
-    checkDependencies(packageConfig.devDependencies)
+    checkDependencies(packageConfig.devDependencies) ||
+    checkDependencies(packageConfig.optionalDependencies)
```

Comment:
I'm not sure that we need to checkout **optionalDependencies**

Reply:
the other option would be to simply return "playwright" if packageConfig.name === "jest-playwright-preset". Because without that the tests are failing

Reply:
@mxschmitt oh! now I see we need it to pass tests. Yeah! We need to find out some workaround it